### ISSUE TITLE
Swift 1.2

### ISSUE THREAD

**Issue by jonas-andersson-stratsys** [dongri/OAuthSwift#47]
Swift 1.2 support would be awesome!

**Comment by tijee** [comment]
+1 :+1:

**Comment by dongri** [comment]
https://github.com/dongri/OAuthSwift/pull/28

**Comment by tijee** [comment]
When do you plan to release it? The last release does not include Swift 1.2 support...

Thanks again :+1:

**Comment by dongri** [comment]
I'm sorry, now I've just released v0.3.3 (Support Swift 1.2)

Carthage: 

```
github "dongri/OAuthSwift" ~> 0.3.3
```

CocoaPods:

```
pod "OAuthSwift", "~> 0.3.3"
```

**Comment by tijee** [comment]
Great, thank you!!
